src/bios_handler.cpp

```diff
@@ -101,11 +101,10 @@ void IbmBiosHandler::biosAttributesCallback(sdbusplus::message_t& i_msg)
             if (auto l_val = std::get_if<std::string>(
```

Comment:
As per D-bus interfaces yaml file your BiosBaseTableType should be somthing like this :

using BiosBaseTableType =std::map<std::string, BiosProperty>;

But why your type is something like 

using BiosBaseTable = std::variant<std::map<std::string, BiosProperty>>;
using BiosBaseTableType = std::map<std::string, BiosBaseTable>; ??


https://github.com/ibm-openbmc/phosphor-dbus-interfaces/blob/1110/yaml/xyz/openbmc_project/BIOSConfig/Manager.interface.yaml#L64C18-L65C44

Reply:
This is the way the message is bundled up in the call back message.
If you see in the code, the type is used to read the callback message sent by the DBus. Had there been any mismatch, it would have failed.

Reply:
okay, can you please tell me what will be stored in std::string mentioned in BiosBaseTableType and BiosBaseTable?

Reply:
As these changes are beyond the scope of this PR.  Can we keep this understanding part of the discussion offline?